Tests/fixtures/classification.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\ClassificationBundle\Model;
+
+use Sonata\CoreBundle\Model\ManagerInterface;
+
+interface CategoryInterface
+{
+    public function setContext($context);
+
+    public function setName($name);
+
+    public function setEnabled($enabled);
+
+    public function setPosition($position);
+}
+
+interface ContextInterface
+{
+    public function setId($id);
+
+    public function setName($name);
+
+    public function setEnabled($enabled);
+}
+
+interface ContextManagerInterface extends ManagerInterface
+{
+}
```

Comment:
Oh right, that's a bit weird, but… can be done by simply tweaking the `autoload-dev` key, right?